Lib/test/test_future.py

```diff
@@ -74,6 +76,16 @@ def test_badfuture10(self):
             from test import badsyntax_future10
         self.check_syntax_error(cm.exception, "badsyntax_future10", 3)
 
+    def test_ensure_flags_dont_clash(self):
```

Comment:
Please add a comment to explain which constants you test here. I understand that you test CO_xxx constants (get from the __future_ module) and PyCF_xxx constants (get from the ast module). Mention also bpo-39562 in the comment.